tools/doctor

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Peforms basic checks for tools required by Magic Modules.
+
+set -e
+
+if [ -t 1 ]; then
+    red=$'\e[1;31m'
+    grn=$'\e[1;32m'
+    yel=$'\e[1;33m'
+    blu=$'\e[1;34m'
+    mag=$'\e[1;35m'
+    cyn=$'\e[1;36m'
+    end=$'\e[0m'
+else
+    red=''
+    grn=''
+    yel=''
+    blu=''
+    mag=''
+    cyn=''
+    end=''
+fi
+
+warn() {
+    printf "%s\n" "${!1}${2}${end}"
+}
+
+found=0
+count_in_path() {
+    echo "Check for ${1} in path..."
+    found=$(which -a "${1}" | uniq | wc -l)
+}
+
+count_in_path "rbenv"
```

Comment:
`rbenv` isn't strictly necessary for running Magic Modules / Ruby. Also, many people use RVM instead of rbenv. I personally like rbenv more, but I seem to be in the minority?

Reply:
Yeah, it's just a warning. I figure if someone already has rvm installed they can ignore it since they already know the difference (and if not it's a good learning opportunity :-)